src/test/java/sg/dex/starfish/samples/IrisSample.java

```diff
@@ -19,7 +19,7 @@
 	public static void main(String... args) {
 		RemoteAgent surfer = RemoteAgentConfig.getRemoteAgent();
 
-		ResourceAsset iris=ResourceAsset.create("assets/iris.csv");
+		ResourceAsset iris=ResourceAsset.create("assets/iris.csv",false);
```

Comment:
If you do tests like this, add assertions to check that there is no content hash added.